examples/react-gatsby/package.json

```diff
@@ -38,11 +38,9 @@
     "@types/react-helmet": "^6.0.0",
     "gatsby-plugin-sass": "^4.8.0",
     "gh-pages": "^3.2.3",
-    "node-sass": "^6.0.0",
-    "node-sass-package-importer": "^5.3.2",
```

Comment:
We don't need `node-sass` anymore, and it's a pain to install (requires the "right" version of python).